src/BiomeJsBinary.php

```diff
@@ -182,4 +173,26 @@ public static function getBinaryName(): string
             default => throw new \Exception(sprintf('Unknown platform or architecture (OS: %s, Machine: %s).', $os, $machine)),
         };
     }
+
+    /**
+     * Whether the current PHP environment is using musl libc.
+     * This is used to determine the correct Biome.js binary to download.
+     */
+    private static function isMusl(): bool
+    {
+        static $isMusl = null;
+
+        if (null !== $isMusl) {
+            return false;
+        }
```

Comment:
Should not it return $isMusl instead of false ?
It would return false even if $isMusl is true.

Reply:
@ralauge-aramis (who does not speak too much :p ) has tested with "return false", and it fails :/

With "return $isMusl", it works as expected :)

Reply:
Oops yes, it should be `return $isMusl`! Thanks :)